### PR TITLE
kube-up: Refactor SSL cert helpers into common

### PR DESCRIPTION
**Pull request by justinsb**
Next step in the the GCE/AWS harmonization

### CODE REVIEW

**Comment by justinsb** [comment]
Going to create a tracking issue for this

**Comment by justinsb** [comment]
@k8s-bot test this flake issue #20646

File: cluster/common.sh [hunk 1]
```diff
@@ -629,3 +629,75 @@ function sha1sum-file() {
...
+#   MASTER_KEY_BASE64
+#   KUBELET_CERT_BASE64
+#   KUBELET_KEY_BASE64
+#   KUBECFG_CERT_BASE64
+#   KUBECFG_KEY_BASE64
+function create-certs {
+  local -r cert_ip="${1}"
+
+  # Determine extra certificate names for master
+  local octets=($(echo "$SERVICE_CLUSTER_IP_RANGE" | sed -e 's|/.*||' -e 's/\./ /g'))
```
Comment (brendandburns) [inline cluster/common.sh]: `${SERVICE_CLUSTER_IP_RANGE}`

**Comment by brendandburns** [comment]
One minor nit, feel free to self LGTM when fixed.

**Comment by justinsb** [comment]
Thanks brendan!  LGTM-ing it.

**Comment by justinsb** [comment]
@k8s-bot test this github flake: #20646

**Comment by justinsb** [comment]
@k8s-bot test this github flake: #20547

**Comment by justinsb** [comment]
Oddly, this isn't in the submit queue either... Going to try a slower close & reopen

**Pull request merged**